src/Utils/AdminUtil.php

```diff
@@ -7,6 +7,7 @@
 class AdminUtil extends API {
     public function getImageJSON($publicId, $resourceType)
     {
-        return $this->getResource($publicId, $resourceType);
+        trigger_error('getImageJSON() is deprecated, use Helper::get_processed_resource($publicId, $resourceType) instead', E_USER_DEPRECATED);
+        return $this->getResourceData($publicId, $resourceType);
```

Comment:
As you're telling them to use `Helper::get_processed_resource()` you should probably be calling that yourself

Reply:
HA! good point.